src/GraphManager/hooks/useLogoutUser.ts

```diff
@@ -0,0 +1,37 @@
+import { gql, useMutation } from "@apollo/client";
+import { ApolloQueryResponse } from "./types";
+
+const LOGOUT = gql`
+  mutation logout {
+    logout {
+      Message
+    }
+  }
+`;
+
+export interface LogoutFn {
+  (): Promise<LogoutUserFnResponse>;
+}
+
+export interface LogoutUserFnResponse {
+  data?: {
+    logout: null;
```

Comment:
This interface doesn't match the query's return type, it should use the `Status` type from hooks/types.ts for consistency. Since error messages will arrive in the Message field, as for all other queries that return a `Message`.